mp/src/materialsystem/stdshaders/readme.txt

```diff
@@ -0,0 +1,12 @@
+Install Strawberry Perl and add it to your system path variable.
+
+Open CMD and type:
+perl -MCPAN -e shell
+
+Then in the cpan shell type:
+install String::CRC32
+
+Edit the paths in buildmomentumshaders.bat to fit your setup.
+
+After all that, you should be able to run buildmomentumshaders.bat successfully
+Building PBR can take like half an hour, good luck.
```

Comment:
There already is a readme in this folder, under the `README.md` file. Edit that instead.

Reply:
I didn't wanna mess with it because I didn't make it, but I can try to combine them into one readme if you insist

Reply:
Alrighty